### ISSUE TITLE
CTRL+C

### ISSUE THREAD

**Issue by Zavy86** [curzon01/mqtt2sql#3]
Hi, I can't stop the program...

**Comment by curzon01** [comment]
which version, which OS

**Comment by Zavy86** [comment]
Hi, is a debian 8 lastest version 1.5.0022

Output:
```
^CDEBUG:__main__:Sending CONNECT (u1, p1, wr0, wq0, wf0, c1, k60) client_id=mqtt2sql.py-4116
DEBUG:__main__:Received CONNACK (0, 0)
DEBUG:__main__:Sending SUBSCRIBE (d0, m3) [('#', 0)]
DEBUG:__main__:Received SUBACK
DEBUG:__main__:Received PUBLISH (d0, q0, r1, m0), 'tele/SONOFF_ICT/LWT', ...  (6 bytes)
DEBUG:__main__:Received PUBLISH (d0, q0, r1, m0), 'cmnd/SONOFF_ICT/POWER', ...  (3 bytes)

```

**Comment by curzon01** [comment]
That's currently a restriction of the mqtt library loop() function

Workaround is to kill the PID which runs the process (e.g. `kill -9 $(pgrep -f "python.*mqtt2sql.py")`) or using the suggested [systemd daemon](https://github.com/curzon01/mqtt2sql#systemd-manager-daemon-example)

**Comment by Zavy86** [comment]
ok thank you!

**Comment by gsantner** [comment]
I used kill9 as well, but cant you catch the break and call sys.exit(0) inside python?

**Comment by gsantner** [comment]
btw systemd restart is hanging for me as well, I have to force kill also at systemd

**Comment by curzon01** [comment]
should be fixed using latest (v2.1.0031)
